core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -472,7 +472,15 @@ ExWorkProcRetcode ExHbaseAccessInsertSQTcb::work()
 	  {
 	    rc = applyPred(scanExpr());
 	    if (rc == 1) // expr is true or no expr
-	      step_ = CREATE_MUTATIONS;
+	      {
+		 rc = evalInsDelPreCondExpr();
```

Comment:
Should we consider move this expression evaluation to SETUP_INSERT step since it evaluates from queue entry.